### PR TITLE
[YUNIKORN-2571] Add hierarchy icon to queue node

### PR DESCRIPTION
**Pull request by doupache**
### What is this PR for?
Add hierarchy icon to queue node
icon from [svgrepo](https://www.svgrepo.com/svg/497190/hierarchy) which is MIT License

image

### What type of PR is it?
* [x] - Improvement

### What is the Jira issue?
[YUNIKORN-2571](https://issues.apache.org/jira/browse/YUNIKORN-2571)

### Screenshots (if appropriate)

image

### CODE REVIEW

File: src/app/components/queue-v2/queues-v2.component.ts [hunk 1]
```diff
@@ -159,6 +159,13 @@ function queueVisualization(rawData : QueueInfo){
           .attr("height", 30)
           .attr("fill", "#e6f4ea")
           .attr("class", "cardBottom");
+        
+        group.append("image")
+        .attr("href", "./assets/images/hierarchy.svg") 
+        .attr("x", 5) 
+        .attr("y", 5) 
+        .attr("width", 20)
+        .attr("height", 20);
```
Comment (baconYao) [inline src/app/components/queue-v2/queues-v2.component.ts:168]:
```suggestion
        group.append("image")
          .attr("href", "./assets/images/hierarchy.svg") 
          .attr("x", 5) 
          .attr("y", 5) 
          .attr("width", 20)
          .attr("height", 20);
```

Comment (baconYao) [reply src/app/components/queue-v2/queues-v2.component.ts:168]: Should we care about the indent like other group here?

**Review by baconYao — COMMENTED** [review]
Please see my suggestion, thanks.

File: src/app/components/queue-v2/queues-v2.component.ts (hunk 1 above)
Comment (doupache) [reply src/app/components/queue-v2/queues-v2.component.ts]: Thanks @baconYao.

**Comment by ryankert01** [comment]
+1, Nice icon.

**Comment by chenyulin0719** [comment]
Hi @wilfred-s, @craigcondit,

What else we need to do to include the image from MIT license? 

The [MIT lincese](https://opensource.org/license/mit) says "The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software."

**Comment by wilfred-s** [comment]
We need to add a couple of lines to the License file. Like we did in the k8shim LICENSE file:
```
--------------------------------------------------------------------------------
This product bundles various third-party components under other open source
licenses. This section summarizes those components and their licenses.

MIT License
------------
src/assets/images/hierarchy.svg
```

Besides that the repo has the following comment:
https://www.svgrepo.com/page/licensing/
So we can add a text to the about box which we already need for our 3rd party licenses to make them easily accessible. This was doen in YUNIKORN-2448 so we should be able to piggy-back on that one

**Comment by doupache** [comment]
I have updated the SVGRepo licensing info.
cc @wilfred-s 

image

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Pull request closed without merging**